{{cookiecutter.project_slug}}/HELP.md

```diff
@@ -50,4 +50,133 @@ neuro-flow run jupyter
 neuro-flow kill jupyter
 ```
 
-* The job with Jupyter Notebooks is terminated. The notebooks are saved on the platform storage. You may run `make download-notebooks` to download them to the local `notebooks/` directory.
+* The job with Jupyter Notebooks is terminated. The notebooks are saved on the platform storage.
+
+### Help
+
+```shell 
+neuro-flow ps
+```
+
+* The list of all available template jobs is printed along with their statuses.
+
+
+## Data
+
+### Uploading to the Storage via Web UI
+
+On local machine, run `neuro-flow run filebrowser` and open the job's URL on your mobile device or desktop.
+Through a simple file explorer interface, you can upload test images and perform file operations.
+
+### Uploading to the Storage via CLI
+
+On local machine run `neuro-flow mkvolumes`. This commands creates storage folders for all defined volumes. You only need to run this once.
+
+After storage folders have been created run `neuro-flow upload data` from local machine as well. This command pushes local files stored in ./data into storage:{{ cookiecutter.project_slug }}/data mounted to your development environment's /project/data.
+
+### Uploading data to the Job from Google Cloud Storage
+
+Google Cloud SDK is pre-installed on all jobs produced from the Base Image.
+
+Please refer to the [documentation](https://docs.neu.ro/toolbox/accessing-object-storage-in-gcp) explaining how to start using GCP with the template.
```

Comment:
these docs will be updated to refer `neuro-extras` usage later?

Reply:
Yes, absolutely. It is in the plans: as soon as `neuro-flow` supports multiple volumes/envvars passing to custom actions